apps/linode-marketplace-passbolt/roles/passbolt/templates/conf.j2

```diff
@@ -0,0 +1 @@
+# rename and add any templates for passbolt
```

Comment:
this file can be removed if not required

Reply:
Removed.